client/src/pages/SearchResultsPage.jsx

```diff
@@ -12,14 +13,15 @@ import PostedOn from "../components/PostedOn";
 
 const mockData = "02/14/2003";
 
-const SearchResultsPage = () => {
+const SearchResultsPage = (props) => {
+  const postedDate = format(new Date(mockData), "M/d/yy");
   return (
     <BasePage>
       <Stack spacing={2}>
         <Typography variant="pageTitle" marginTop="6rem">
           Find Internships
         </Typography>
-        <PostedOn date={mockData} />
+        <PostedOn date={postedDate} />
```

Comment:
I don't think this is going to be the right place to put this right? This component will most likely be used on the InternshipCard and details if i recall. we can probably leave this testing code out until we're ready to implement it